### ISSUE TITLE
Add composer.json

### ISSUE THREAD

**Issue by strarsis** [johngodley/redirection#623]
It would be great if this repository also gets a composer.json for composer support so it can be installed directly using composer, notably beta versions and other versions that hadn't been published on WordPress yet (I am using wpackagist for pulling WordPress plugins using composer).

**Comment by strarsis** [comment]
PR ready: https://github.com/johngodley/redirection/pull/664

**Comment by leopoiroux** [comment]
You can install the plugin with composer 
https://wpackagist.org/search?q=redirection&type=any&search=redirection

**Comment by strarsis** [comment]
@leopoiroux: Right, composer support is meant for installing from master or specific commit for fixes that hadn't been released yet.
